Source/CesiumRuntime/Private/Tests/Cesium3DTileset.spec.cpp

```diff
@@ -18,6 +18,10 @@
 #include <Cesium3DTilesSelection/TilesetSharedAssetSystem.h>
 #include <CesiumAsync/ICacheDatabase.h>
 
+THIRD_PARTY_INCLUDES_START
```

Comment:
Just curious what this macro is used for? (Why does it apply to `Uri.h` but not the other includes form cesium-native here?

Reply:
Arguably we should wrap all the cesium-native headers in that. But it's necessary here because one of the ada-url headers triggers warnings (which become errors) in Unreal builds.